Torch.Server/Views/ModListControl.xaml.cs

```diff
@@ -205,9 +205,9 @@ private void UserControl_MouseMove(object sender, MouseEventArgs e)
             {
                 _hasOrderChanged = true;
                 var modList = (MtObservableList<ModItemInfo>)DataContext;
-                //modList.Move(modList.IndexOf(_draggedMod), modList.IndexOf(targetMod));
-                modList.RemoveAt(modList.IndexOf(_draggedMod));
-                modList.Insert(modList.IndexOf(targetMod), _draggedMod);
+                modList.Move(modList.IndexOf(targetMod), _draggedMod);
```

Comment:
You should ensure that your index is valid before passing it on. If `targetMod` is null, you'll get interesting exceptions.

Reply:
I have already checked if `targetMod` is null, in fact on line L204 there is code that was already implemented to check if `targetMod` was not null.